### PR TITLE
use loadCard for active players

### PR DESCRIPTION
**Pull request by Jamiras**
Updates the active players callback to use the `loadCard` call for the user. Also updates the game card to match the new style (no parentheses on console name).

Before:
image

After:
image

### CODE REVIEW

File: public/js/activePlayersBootstrap.js [hunk 1]
```diff
@@ -72,14 +89,15 @@ var ActivePlayersViewModel = function () {
   };
 
   this.ConvertToObservablePlayer = function (player) {
+    CacheCardDiv('game', player.GameID, player.GameTitle, player.ConsoleName, player.GameIcon);
```
Comment (Jamiras) [inline public/js/activePlayersBootstrap.js:92]: I'm not that familiar with `ko`. Is this going to be problematic?

Comment (luchaos) [reply public/js/activePlayersBootstrap.js:92]: I don't think so. I'm not sure why those are observables looking at it.

File: public/js/activePlayersBootstrap.js [hunk 2]
```diff
@@ -1,3 +1,20 @@
+function CreateCardIconDiv(type, id, title, icon, url) {
```
Comment (luchaos) [inline public/js/activePlayersBootstrap.js:1]: `avatar()` would match the backend variant - not important here though.

**Review by luchaos — APPROVED** [review]

File: public/js/activePlayersBootstrap.js (hunk 1 above)
Comment (luchaos) [reply public/js/activePlayersBootstrap.js:92]: Seems to work just fine for knockout (`ko`).

**Pull request merged**